pkg/helm/charts.go

```diff
@@ -144,13 +144,9 @@ func (c Charts) Vendor() error {
 
 // Add adds every Chart in reqs to the Manifest after validation, and runs
 // Vendor afterwards
-func (c *Charts) Add(reqs []string) error {
+func (c *Charts) Add(reqs []string, vendor bool) error {
```

Comment:
What's the use of that bool argument? Like, what a case where I like to add but not vendor?

Reply:
Tests, but I think I'll setup a fake local helm repo instead. I just don't want to depend on a remote helm chart for tests

Reply:
Just noticed that there was already a helm interface there. Much better